packages/db/configs/sqlite.config.ts

```diff
@@ -4,8 +4,9 @@ import type { Config } from "drizzle-kit";
 dotenv.config({ path: "../../.env" });
 
 export default {
+  dialect: "sqlite",
+  driver: "better-sqlite" as never, // TODO: Remove this line when the issue is fixed
```

Comment:
Which issue are you referring to ?

Reply:
There is currently no issue for this yet, but without defining the driver there is currently an error that you have to define a sqlite driver

Reply:
Could be useful to create one on the drizzle-kit repository so that this gets more visibility

Reply:
Yeah you're right, I'll try it again and create an issue if it was not fixed in 0.21.1

Reply:
Okay the issue got actually fixed, even if nothing about it was mentioned in the release notes